pkg/platform/runtime/setup/setup.go

```diff
@@ -229,21 +229,21 @@ func (s *Setup) updateArtifacts() ([]artifact.ArtifactID, error) {
 		}
 
 		// Retrieve environment definitions for artifact
-		envDef, err := as.EnvDef(unpackedDir)
+		envDef, err := as.EnvDef(tempUnpackedDir)
```

Comment:
This is the only function where we pass the `_runtime_temp` dir directly and I believe for now it will have to stay that way.  There are two implementations of this function, one for the alternative builder and one for camel. The alternative builder just wants the `runtime.json` file while camel relies a bit more on the directory structure.